web/src/pages/api/public/metrics/daily.ts

```diff
@@ -47,73 +47,47 @@ export default withMiddlewares({
               SUM(o.completion_tokens) "outputUsage",
               SUM(o.total_tokens) "totalUsage",
               COALESCE(SUM(o.calculated_total_cost), 0)::DOUBLE PRECISION as "totalCost"
-            FROM
-              traces t
-            LEFT JOIN observations_view o ON o.trace_id = t.id AND o.project_id = t.project_id
+            FROM traces t
+            LEFT JOIN observations_view o
+            ON o.trace_id = t.id
+            AND o.project_id = t.project_id
             WHERE o.start_time IS NOT NULL
+              AND o.project_id = ${auth.scope.projectId}
               AND t.project_id = ${auth.scope.projectId}
               ${traceNameCondition}
               ${userCondition}
               ${tagsCondition}
               ${fromTimestampCondition}
               ${toTimestampCondition}
-            GROUP BY
-              1,
-              2
-            ORDER BY
-              1,
-              2
+            GROUP BY 1, 2
           ),
           daily_model_usage AS (
             SELECT
               "date",
-              json_agg(json_build_object('model',
-                  model,
-                  'inputUsage',
-                  "inputUsage",
-                  'outputUsage',
-                  "outputUsage",
-                  'totalUsage',
-                  "totalUsage",
-                  'totalCost',
-                  "totalCost",
-                  'countObservations',
-                  "countObservations",
-                  'countTraces',
-                  "countTraces")) daily_usage_json
+              sum("countTraces")::integer "countTraces",
+              sum("countObservations")::integer "countObservations",
+              sum("totalCost")::DOUBLE PRECISION "totalCost",      
```

Comment:
This performs essentially the same aggregation by date as the `daily_stats` query did before. The main difference is that we use `observation.start_time` as the baseline for the date instead of `traces.timestamp`. Since we've joined on the column in the end, I believe that this actually improves the situation as we're using a "single date of truth".

Reply:
sounds good, if this is mentioned in the api reference (fern), we need to update this there as well

Reply:
@marcklingen I don't see that fern would need an update. All statements in there are still accurate. Good callout though to double check there 🙏

Reply:
just thought that we might have inlcluded a sentence in the `docs` part of the fern schema to highlight how the filters apply and what values are returned. if we return the generation start_at, we could be more specific there to let people know

Reply:
@marcklingen Added a section in the fern schema to highlight how I intend to use the timestamp provided.